tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AWS/AwsLambdaTests.cs

```diff
@@ -43,6 +46,7 @@ public async Task SubmitsTraces()
                 return;
             }
 
+            EnvironmentHelper.CustomEnvironmentVariables[ConfigurationKeys.TraceDataPipelineEnabled] = dataPipelineEnabled.ToString();
```

Comment:
nit: as we're adding this in a bunch of places, we can probably add a helper to the base `TestHelper` class:

```csharp
protected void SetDataPipelineEnabled(bool enabled)
{
    SetEnvironmentVariable(ConfigurationKeys.TraceDataPipelineEnabled, enabled.ToString());
}
```

And then change this to 

```suggestion
            SetDataPipelineEnabled(dataPipelineEnabled);
```